src/main/scala/no/ndla/imageapi/controller/ImageControllerV2.scala

```diff
@@ -157,6 +158,45 @@ trait ImageControllerV2 {
         )
         responseMessages(response404, response500))
 
+    val getStoredParameters =
+      (apiOperation[StoredParameters]("getStoredParameters")
+        summary "Get stored parameters"
+        notes "Get all stored parameters for an image"
+        parameters(
+        asHeaderParam[Option[String]](correlationId),
+        asPathParam[String](imageUrl))
+        responseMessages(response404, response400, response500))
+
+    get("/storedParameters/:image_url", operation(getStoredParameters)) {
```

Comment:
Jeg vil foretrekke at vi ikke bruker camelCase i path-parametre. Vi har frem til nå benyttet `-` som skille på ord. Dette er også generelt anbefalt av "internett". Er det noe i veien for å bytte dette til `stored-parameters`?

Reply:
Enig, endrer dette.